bedrock/base/templates/includes/protocol/site-footer.html

```diff
@@ -66,6 +66,11 @@ <h5>
             <li><a href="{{ url('privacy.notices.websites') }}#cookies" data-link-type="footer" data-link-name="Cookies">{{ _('Cookies') }}</a></li>
             <li><a href="{{ url('legal.index') }}" data-link-type="footer" data-link-name="Legal">{{ _('Legal') }}</a></li>
           </ul>
+        {% if LANG.startswith('en-') %}
+          <p class="mzp-c-footer-license">
+            {{ _('Visit Mozilla Corporation’s not-for-profit parent, the <a href="%s" data-link-type="footer" data-link-name="Mozilla Foundation">Mozilla Foundation</a>.'|format('https://foundation.mozilla.org')) }}
```

Comment:
I think `data-link-type` and `data-link-name` should be outside of the translation block. We want to be able to measure the performance of the link using the same query no matter which language we're looking at.